java/src/jmri/jmrix/dccpp/dccppovertcp/package.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 <html>
     <head>
+        <title>jmri.jmrix.dcc.dccpp package information</title>
```

Comment:
Not a big issue, just a point in passing, but `package-info.java ` files are [now preferred](https://www.jmri.org/help/en/html/doc/Technical/Javadoc.shtml) to `package.html`files.  I tend to switch the file over when I touch it.

Reply:
There is more listing to do in this package.  I’ll make that change when I get to that.